src/App.js

```diff
@@ -35,33 +38,45 @@ function App() {
     }
   });
   return (
-    <div className="App font-primary sm:text-base text-xs">
+    <div className="App font-primary md:grid md:grid-cols-[20%,_80%] sm:text-base text-xs">
```

Comment:
I noticed that it is not responsive it is too much in medium size screen:
As you can see in following photo. I cannot see the text anymore.

![Screenshot from 2023-04-11 16-46-09](https://user-images.githubusercontent.com/105173981/231131507-64793fbd-364e-4ab4-89e4-98332adc13f5.png)

Reply:
I noticed that grid is not properly functioning in the following photo:

As you can see nav bar is taking more space than expected

![Screenshot from 2023-04-11 16-46-22](https://user-images.githubusercontent.com/105173981/231131787-3d1aa505-dba8-41ba-94fb-47c27acb8070.png)